usefulUtilities/voteApp/appResources/resources/Tablet.html

```diff
@@ -0,0 +1,45 @@
+<html>
+
+<head>
+    <title>Tablet</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css?family=Cairo" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+        crossorigin="anonymous">
+    <link rel="stylesheet" href="css/glyphs/styles.css">
+    <link rel="stylesheet" href="css/modal/modal.css">
+    <link rel="stylesheet" href="css/app.css">
+</head>
+
+<body>
+    <div class="container mt-2" id="app">
+        <!-- App status modals -->
+        <loggedin-modal :loggedIn="dataStore.loggedin"></loggedin-modal>
```

Comment:
Reviewing this file depends on being able to review `Tablet.js`, so I'm holding off for now.